security-force-monitor/sfm-cms
# fix path failure for data import and add downloaded data as phony target

## Overview
See title. Fixes an issue found when trying to import on the staging server.

### Notes
I added the `~/country_data` directory as a phony target so the import cleans up the directory and starts with fresh data each time it runs. Is this the correct way to do it?

## Testing Instructions
- download data:
```
docker-compose run --rm app python manage.py download_country_data \
    --sources_doc_id 1dkGS6Ocyc2YYQ5IopEdjz38t9kaT34XfIAab0iteGoY \
    --location_doc_id 1ztOfGaQT3WDrq-BOjT0x5VErzgrWQ0Ku \
    --entity_doc_id 19Hk6OD5AYjWQCUaTWPzkGm9sLqp4e5v_n667M0CgaQQ \
    --country_code ae \
    --parent_directory sfm_pc/management/commands/country_data
```

- import data:
```
docker-compose run --rm app python manage.py import_country_data \
    --country_code ae \
    --country_path country_data/countries/ae \
    --sources_path country_data/sources.csv
```

the data should import successfully.
